types/react-hyperscript/index.d.ts

```diff
@@ -1,28 +1,145 @@
 // Type definitions for react-hyperscript 3.0
 // Project: https://github.com/mlmorg/react-hyperscript
-// Definitions by: tock203 <https://github.com/tock203>
+// Definitions by: roshal <https://github.com/roshal>
```

Comment:
Did @tock203 ask to be removed? you didn't mention in the PR body

Reply:
@tock203 no longer exists